openslides/core/static/js/core/core.js

```diff
@@ -605,6 +605,46 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.countdowns = [];
+    $scope.messages = [];
+    _.forEach(Projector.get(1).elements, function(element) {
+        if (element.name == 'core/countdown') {
+            $scope.countdowns.push(element);
+        }
+        if (element.name == 'core/message') {
+            $scope.messages.push(element);
+        }
+    });
+    $scope.addCountdown = function () {
+        $http.post(
+            '/rest/core/projector/1/activate_elements/',
+            [{name: 'core/countdown', status: 'stop', visible: true,
+                countdown_time: 42, stable: false}]
+            // TODO countdown_time: config auslesen
+        ).success(function(data) {
+            $scope.countdowns = [];
+            _.forEach(data.elements, function(element) {
+                if (element.name == 'core/countdown') {
+                    $scope.countdowns.push(element);
+                }
+            });
+        });
+    };
+    $scope.removeCountdown = function (countdown) {
+        // TODO
+    };
+    var stopped;
+    $scope.startCountdown = function (countdown) {
+        stopped = $timeout(function() {
+            countdown.countdown_time--;
+            $scope.startCountdown(countdown);
+        }, 1000);
+    };
+    $scope.stopCountdown = function(){
+       $timeout.cancel(stopped);
+    };
```

Comment:
Currently I can start and stop the countdown locally (without any notice to the projector). What and when the projector need the update request?